sdf.md

```diff
@@ -263,15 +265,29 @@ Declaration:
   enclosing definition.  Every declaration can also be used as a
   definition for reference in a different place.
 
+SDF Document:
+: Container for SDF Definitions, together with data
+  about the SDF Document itself (information block).
+  Represented as a JSON text representing a single JSON map, which is
+  built from nested maps.
+
+SDF Model:
+: Definitions and declarations that describe the digital interaction
+  opportunities offered by one or more classes of Things, represented
```

Comment:
Is "capabilities" maybe a better fit than "opportunities" here?

Reply:
I like "opportunities", as that is in line with the "affordance" style of thinking, but "capabilities" would work as well, even if this is loaded with a different bag of connotations (why I don't prefer it).

Reply:
I see, that makes sense, thank you :)